web-common/src/features/dashboards/leaderboard/Leaderboard.svelte

```diff
@@ -221,6 +222,7 @@
       timeRange,
       comparisonTimeRange,
       measures,
+      limit: (maxValuesToShow - aboveTheFold.length).toString(),
```

Comment:
I'm wondering if we'd get a `400` if `limit` were 0 or negative. Maybe guard against `limit <= 0` in the `enabled` clause?

Reply:
It shouldnt happen, but doesnt hurt to add a safeguard.